File: algorithms/interview-preparation-kit-1/js/code-response/zeroMeansZero/zeroMeansZero.test.js
import { zeroMeansZero } from './zeroMeansZero.js';

describe('String Utils', () => {
  test('remove zero of 7 + 8', () => {
    expect(zeroMeansZero(7, 8)).toBe('15');
  });

  test('remove zero of 15 + 5', () => {
    expect(zeroMeansZero(15, 5)).toBe('2');
  });

  test('remove zero of 99 + 6', () => {
    expect(zeroMeansZero(99, 6)).toBe('15');
  });
});

File: algorithms/interview-preparation-kit-1/ruby/code/zeroMeansZero/spec/zeroMeansZero.test.rb
require_relative "../src/zeroMeansZero"

describe ZeroMeansZero do
    describe ".noZero" do
        context "Sum without zero" do
            it "return 5" do
                expect(ZeroMeansZero.noZero(5,45)).to eql(5)
            end
        end
    end
end